Documentation/README_TapToPlace.md

```diff
@@ -0,0 +1,95 @@
+# Tap to Place
+
+![TapToPlace](Images/Solver/TapToPlace/TapToPlaceIntroGif.gif)
+
+Tap to Place is a far interaction component that is used to place a game object on surface. This component is useful for placing objects on a spatial mesh. Tap to Place uses a combination of two clicks and head movement to place an object. A click to start the placement, head movement to change the position of the object and a click to place.
+
+## Using Tap to Place
+
+1. Set up the scene
+    - Create a new unity scene
+    - Add MRTK to the scene by navigating to the Mixed Reality Toolkit menu > Add to Scene and Configure
+    - Add a cube to the scene and change the scale to 0.2 and change the position to (0, 0, 0.7).
+1. Attach [Tap to Place](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.Utilities.Solvers.TapToPlace.html) to a game object with a collider
```

Comment:
this should be an xref

Reply:
xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.SolverHandler
replace with namespace of taptoplace

Reply:
added